### PR TITLE
Fixes for Web Tests

### PR DESCRIPTION
**Pull request by cryptoquick**
Some misc stuff needed to fix the web tests.

- [x] Fix swapped mnemonic arguments.
- [x] Add NODE_HOST env var, and methods to switch the NODE_HOST at runtime.
- [x] Import cleanup.

### CODE REVIEW

File: src/data/constants.rs [hunk 1]
```diff
@@ -77,13 +77,14 @@ pub async fn switch_network(network_str: &str) -> Result<()> {
...
-pub static LIST_ASSETS_ENDPOINT: Lazy<String> = Lazy::new(|| dot_env("LIST_ASSETS_ENDPOINT"));
-pub static SEND_ASSETS_ENDPOINT: Lazy<String> = Lazy::new(|| dot_env("SEND_ASSETS_ENDPOINT"));
-pub static ACCEPT_TRANSFER_ENDPOINT: Lazy<String> =
-    Lazy::new(|| dot_env("ACCEPT_TRANSFER_ENDPOINT"));
-pub static VALIDATE_TRANSFER_ENDPOINT: Lazy<String> =
-    Lazy::new(|| dot_env("VALIDATE_TRANSFER_ENDPOINT"));
+pub static NODE_HOST: Lazy<AsyncRwLock<String>> =
+    Lazy::new(|| AsyncRwLock::new(dot_env("NODE_HOST")));
+
+pub async fn get_url(path: &str) -> String {
```
Comment (grunch) [inline src/data/constants.rs]: Calling this function `get_endpoint()` would be more descriptive

Comment (cryptoquick) [reply src/data/constants.rs]: Good suggestion! Done!

**Review by grunch — APPROVED** [review]
LGTM! 😃

**Comment by cryptoquick** [comment]
tyvm for the review @grunch :pray:

**Pull request merged**